06-aks-cluster.md

```diff
@@ -4,134 +4,123 @@ Now that the [cluster prequisites and shared Azure service instances are provisi
 
-      ```bash
-      cat sp.json
-      ```
+        > :bulb:
+        >
+        > - Use the env var value of `AKS_INGRESS_CONTROLLER_CERTIFICATE_BASE64`
+        > - Ideally fetching this secret from a platform-managed secret store such as [Azure Key Vault](https://github.com/marketplace/actions/azure-key-vault-get-secrets)
 
-   1. Create `APP_GATEWAY_LISTENER_CERTIFICATE_BASE64` secret in your GitHub repository. For more
-      information, please take a look at [Creating encrypted secrets for a repository](https://docs.github.com/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository).
+        ```bash
+        echo $AKS_INGRESS_CONTROLLER_CERTIFICATE_BASE64
+        ```
 
-      > :bulb:
-      >
-      > - Use the env var value of `APP_GATEWAY_LISTENER_CERTIFICATE_BU0001A004203` and `APP_GATEWAY_LISTENER_CERTIFICATE_BU0001A004204`
-      > - Ideally fetching this secret from a platform-managed secret store such as [Azure KeyVault](https://github.com/marketplace/actions/azure-key-vault-get-secrets)
+    1.  Copy the GitHub workflow file into the expected directory
 
-      ```bash
-      echo $APP_GATEWAY_LISTENER_CERTIFICATE_BU0001A004203
-      echo $APP_GATEWAY_LISTENER_CERTIFICATE_BU0001A004204
-      ```
+        ```bash
+        mkdir -p .github/workflows
+        cat github-workflow/aks-deploy.yaml > .github/workflows/aks-deploy.yaml
+        ```
 
-   1. Create `AKS_INGRESS_CONTROLLER_CERTIFICATE_BASE64` secret in your GitHub repository. For more information, please take a look at [Creating encrypted secrets for a repository](https://docs.github.com/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository).
+    1.  Generate cluster parameter file per region
 
-      > :bulb:
-      >
-      > - Use the env var value of `AKS_INGRESS_CONTROLLER_CERTIFICATE_BASE64`
-      > - Ideally fetching this secret from a platform-managed secret store such as [Azure Key Vault](https://github.com/marketplace/actions/azure-key-vault-get-secrets)
+        ```bash
+        #Region1
+        cat ./azuredeploy.parameters.eastus2.json | \
+        sed "s#<cluster-spoke-vnet-resource-id>#${RESOURCEID_VNET_BU0001A0042_03}#g" | \
+        sed "s#<tenant-id-with-user-admin-permissions>#${TENANTID_K8SRBAC}#g" | \
+        sed "s#<azure-ad-aks-admin-group-object-id>#${AADOBJECTID_GROUP_CLUSTERADMIN}#g" | \
+        sed "s#<log-analytics-workspace-id>#${LOGANALYTICSWORKSPACEID}#g" | \
+        sed "s#<container-registry-id>#${CONTAINERREGISTRYID}#g" | \
+        sed "s#<acrPrivateDns-zones-id>#${ACRPRIVATEDNSZONESID}#g" | \
+        sed "s#<subdomain-name>#${CLUSTER_SUBDOMAIN_03}#g" \
```

Comment:
I think we should change this to be resolved from the `cluster-stamp` instead if that were possible:

`reference(resourceId(subscription().subscriptionId,variables('vNetResourceGroup'),'Microsoft.Network/publicIpAddresses',concat('pip-',variables('appId')))).properties.dnsSettings.domainNameLabel`

Reply:
done